### PR TITLE
x11-misc/trayer-srg: Bump EAPI to version 8

### PR DESCRIPTION
**Pull request by szczot3k**
Package-Manager: Portage-3.0.30, Repoman-3.0.3
Signed-off-by: Filip Kszczot <filip@kszczot.pl>

### CODE REVIEW

File: x11-misc/trayer-srg/trayer-srg-1.1.8-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+DESCRIPTION="trayer fork with multi monitor support and cleaned up codebase"
+HOMEPAGE="https://github.com/sargon/trayer-srg"
+SRC_URI="https://github.com/sargon/${PN}/archive/${P/-srg/}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="${RDEPEND}
```
Comment (thesamesam) [inline x11-misc/trayer-srg/trayer-srg-1.1.8-r1.ebuild:16]: DEPEND is now blank apart from virtual/pkgconfig (which should be in BDEPEND).

Here, DEPEND should be RDEPEND (and defined after, or swap the variable names), and put BDEPEND last with BDEPEND="virtual/pkgconfig"

File: x11-misc/trayer-srg/trayer-srg-1.1.8-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	x11-libs/gdk-pixbuf:2
+	x11-libs/gtk+:2
+	x11-libs/libX11
+"
+BDEPEND=""
+
+S="${WORKDIR}"/${PN}-trayer-${PV}
+
+src_configure() {
+	econf --prefix="${EPREFIX}"
```
Comment (thesamesam) [inline x11-misc/trayer-srg/trayer-srg-1.1.8-r1.ebuild:29]: Are you sure this is OK? Didn't the old comment say it *wasn't* autoconf?

Comment (szczot3k) [reply x11-misc/trayer-srg/trayer-srg-1.1.8-r1.ebuild:29]: Turns out that it is in fact correct, and I'm not sure why it installed without any problems on my test box.

File: x11-misc/trayer-srg/trayer-srg-1.1.8-r1.ebuild (hunk 1 above)
Comment (szczot3k) [reply x11-misc/trayer-srg/trayer-srg-1.1.8-r1.ebuild:16]: Fixed

File: x11-misc/trayer-srg/trayer-srg-1.1.8-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+LICENSE="MIT GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	x11-libs/gdk-pixbuf:2
+	x11-libs/gtk+:2
+	x11-libs/libX11
+"
```
Comment (juippis) [inline x11-misc/trayer-srg/trayer-srg-1.1.8-r1.ebuild:20]:
```
 * VDB: detected possibly incorrect RDEPEND (x11-misc/trayer-srg-1.1.8-r1)
 *  > dev-libs/glib:2
```

**Review by juippis — APPROVED** [review]
```
 * QA Notice: Files built without respecting LDFLAGS have been detected
 *  Please include the following list of files in your report:
 * /usr/bin/trayer-srg
 * QA Notice: Pre-stripped files found:
 * /usr/bin/trayer-srg
```

File: x11-misc/trayer-srg/trayer-srg-1.1.8-r1.ebuild [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 Gentoo Authors
```
Comment (juippis) [inline x11-misc/trayer-srg/trayer-srg-1.1.8-r1.ebuild:1]: Please also keep the old copyright header since the ebuild hasn't changed that much.

**Pull request closed without merging**